instrumentation/akka-http-10.0/src/main/java/io/opentelemetry/auto/instrumentation/akkahttp/AkkaHttpServerTracer.java

```diff
@@ -17,25 +17,28 @@
 package io.opentelemetry.auto.instrumentation.akkahttp;
 
 import akka.http.scaladsl.model.HttpRequest;
-import akka.http.scaladsl.model.HttpResponse;
-import io.opentelemetry.OpenTelemetry;
-import io.opentelemetry.auto.bootstrap.instrumentation.decorator.HttpServerDecorator;
-import io.opentelemetry.trace.Tracer;
+import io.grpc.Context;
+import io.opentelemetry.auto.bootstrap.instrumentation.decorator.HttpServerTracer;
+import io.opentelemetry.context.propagation.HttpTextFormat.Getter;
 import java.net.URI;
 import java.net.URISyntaxException;
 
-public class AkkaHttpServerDecorator
-    extends HttpServerDecorator<HttpRequest, HttpRequest, HttpResponse> {
-  public static final AkkaHttpServerDecorator DECORATE = new AkkaHttpServerDecorator();
-
-  public static final Tracer TRACER =
-      OpenTelemetry.getTracerProvider().get("io.opentelemetry.auto.akka-http-10.0");
+public class AkkaHttpServerTracer extends HttpServerTracer<HttpRequest, HttpRequest, HttpRequest> {
```

Comment:
to self doc that it doesn't use storage
```suggestion
public class AkkaHttpServerTracer extends HttpServerTracer<HttpRequest, HttpRequest, Void> {
```